src/components/GroupsSelect/GroupsSelect.jsx

```diff
@@ -118,12 +118,20 @@ export class GroupsSelectClass extends React.Component {
     }
   }
 
+  handleChange = group => {
+    const { onChange } = this.props
+    try {
+      onChange(group)
+    } catch (error) {
+      throw new Error(`There was a problem when selecting a group : ${error}`)
```

Comment:
Here, we lose stack trace information from the original error :/ To me, it does not serve any purpose to rethrow the error. We shoud:

- Send the error to sentry
- Alert the user that there was an error
- If we throw a higher level error, it should be chained to the original error. 

```js
catch (originalError) {
  const error = new Error(`There was a problem when selecting a group : ${error}`)
  error.originalError = originalError
  throw error
}
```

Reply:
Actually, I do not know precisely how to properly chain error so that they surface in Sentry. Here I think I'd throw the original error.

Reply:
après discussion try/catch déplacé sur le parent ContactInfo. L'enfant GroupSelect ne devrait pas avoir à à gérer ces erreurs qui sont hors de son contexte.